cpp/src/parquet/encoding_benchmark.cc

```diff
@@ -452,6 +452,134 @@ BENCHMARK(BM_ByteStreamSplitEncode_Float_Avx512)->Range(MIN_RANGE, MAX_RANGE);
 BENCHMARK(BM_ByteStreamSplitEncode_Double_Avx512)->Range(MIN_RANGE, MAX_RANGE);
 #endif
 
+template <typename DType, typename NumberGenerator>
+static void BM_DeltaBitPackingEncode(benchmark::State& state, NumberGenerator gen) {
+  using T = typename DType::c_type;
+  std::vector<T> values = gen(state.range(0));
+  std::vector<uint8_t> output(state.range(0) * sizeof(T), 0);
+  auto encoder = MakeTypedEncoder<DType>(Encoding::DELTA_BINARY_PACKED);
+  for (auto _ : state) {
+    encoder->Put(values.data(), static_cast<int>(values.size()));
+    encoder->FlushValues();
+  }
+  state.SetBytesProcessed(state.iterations() * values.size() * sizeof(T));
```

Comment:
Also call `state.SetItemsProcessed`?

Reply:
Ping.

Reply:
I can add it, but I wonder if we really needs it, isn't it just `BytesProcessed in bytes / sizeof(T) `?

Reply:
Right, but the output is more useful like that.